eval/src/vespa/eval/streamed/streamed_value_utils.h

```diff
@@ -66,7 +66,7 @@ class LabelBlockStream {
                      uint32_t num_mapped_dims)
       : _num_subspaces(num_subspaces),
         _labels(label_buf),
-        _subspace_index(-1),
```

Comment:
I thought about using a common npos since one is defined in the LabelBlock, but since you are index based, 'num_spubspaces' would be appropriate here (equivalent with 'end()')

Reply:
good idea, thanks